plugins/core/webview/src/q-ui/components/reauth.vue

```diff
@@ -2,57 +2,92 @@
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
 <template>
-    <div class="font-amazon bottom-small-gap title centered">{{ title }}</div>
-    <div class="font-amazon bottom-small-gap title centered">Please re-authenticate to continue</div>
-
-    <button
-        class="login-flow-button continue-button font-amazon"
-        v-on:click="handleContinueClick()"
-    >
-        Re-authenticate
+    <button v-if="!this.authenticating && this.app === 'TOOLKIT'" class="back-button" @click="back" tabindex="-1">
+        <svg width="24" height="24" viewBox="0 -3 13 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path
+                d="M4.98667 0.0933332L5.73333 0.786666L1.57333 4.94667H12.0267V5.96H1.57333L5.73333 10.0667L4.98667 10.8133L0.0266666 5.8V5.10667L4.98667 0.0933332Z"
+                fill="#21A2FF"
+            />
+        </svg>
```

Comment:
Will this again generate the same issue that has two back buttons?

Reply:
nope, `reauth.vue` now is independent to `login.vue`

Reply:
so in `root.vue`
```
<template>
    <Login v-if>
    <Reauth v-else>
</template>
```

Reply:
backbutton lives within Login and Reauth respectively

Reply:
got it